k8s/node.go

```diff
@@ -56,3 +59,13 @@ func (n *Node) GetURI() string {
 func (n *Node) GetK8sObject() *k8sapi.Avalanchego {
 	return n.k8sObj
 }
+
+// GetStakingURL
+func (n *Node) GetStakingURL() string {
+	return fmt.Sprintf("%s:%d", n.k8sObj.Status.NetworkMembersURI[0], constants.DefaultStakingPort)
```

Comment:
Can we just use `n.uri` instead of `n.k8sObj.Status.NetworkMembersURI[0]`?

Reply:
We could, although the `Status` object holds the actual response from k8s. `n.uri` _should_ contain the same....

Reply:
I think it'd be cleaner to use `n.uri`. If it ever happens that `n.uri` !=  `n.k8sObj.Status.NetworkMembersURI[0]` then something is wrong and we'd want to know